results2junit

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
+#
+# results2junit - convert docker-autotest results file to Jenkins format
+#
+"""
+Convert docker-autotest results file (named 'status') to a results.junit
+file suitable by Jenkins.
+"""
+
+import argparse
+import os
+import re
+import sys
+import time
+
+
+def xml_escape(input_string):
+    """
+    Escape XML-sensitive characters in a string
+    """
+    if not input_string:
+        return ''
+    if isinstance(input_string, basestring):
+        s_out = input_string
+    else:
+        s_out = '%s' % input_string
+
+    s_out = s_out.replace('&', '&amp;')
+    s_out = s_out.replace('<', '&lt;')
+    s_out = s_out.replace('>', '&gt;')
+    s_out = s_out.replace('"', '&quot;')
+    return s_out
+
+
+class TestSuite(object):
+    """
+    Set of test results, initialized from a list read by blah blah
+    """
+
+    def __init__(self, name, results):
+        self.input_name = name
+        # Strip gobbledygook from rhel-latest-7-3-ADEPT-full-26-a451b130bf40
+        self.name = re.sub(r'-\d+-[0-9a-f]{11,}$', '', name)
+        self.results = results
+        self.count = {'tests': 0, 'failures': 0, 'errors': 0, 'skipped': 0}
```

Comment:
It's presumptuous at this level to try and sanitize the name based on any kind of format not tied to this code or project.  At best It's  obfuscating a legitimate name (erasing clues to what may have gone wrong).  At worst, it causes results to clash because the name becomes common vs some other host.  Best to just take what the caller provided. 

NOTE: [I've fixed the long ugly names being passed by ADEPT](https://github.com/cevich/autotest-docker-enabled-product-testing/commit/d31ad6875a50f614ff22efc6f8b1c6214bc26573)